api/cinema/cinemaRoute.js

```diff
@@ -1 +1,54 @@
 //cinema endpoints
+const express = require('express');
+const router = express.Router();
+const Cinema = require('./Cinema');
+
+router.get('/', async (req, res) => {
+  const cinemas = await Cinema.find();
+  return res.status(200).json(cinemas);
+});
+
+router.post('/', async (req, res) => {
```

Comment:
This should be only availabla for admin, so we should add adminMiddleware between the route path and the async callback